src/main/java/xbot/common/command/BaseCommand.java

```diff
@@ -44,7 +47,16 @@ public String getPrefix() {
     }
 
     @Override
-    public abstract void initialize();
+    public void initialize() {
+        // the name might not be set at construction, so let's update it here
+        this.runningAlert.setText(this.getName());
+        this.runningAlert.set(true);
+    }
+
+    @Override
+    public void end(boolean isInterrupted) {
+        this.runningAlert.set(false);
```

Comment:
We notably need to make sure that child classes remember to call super. I don't have a non-complicated way of enforcing that